packages/uicore/src/components/GroupedThumbnailSelect/GroupedThumbnailSelect.tsx

```diff
@@ -34,7 +34,7 @@ export interface GroupedThumbnailSelectProps extends Omit<ThumbnailSelectProps,
   groups: Group[]
 }
 
-const GroupedThumbnailSelect: React.FC<ConnectedGroupedThumbnailSelectProps> = props => {
+export function GroupedThumbnailSelect(props: ConnectedGroupedThumbnailSelectProps): JSX.Element {
```

Comment:
why do we need to remove React.FC?

Reply:
There are downsides to using React.FC - https://github.com/facebook/create-react-app/pull/8177. It was being used earlier across several apps, but it is better to explicitly define the return types.